### ISSUE TITLE
Support interoperability with Tiled

### ISSUE THREAD

**Issue by ftomassetti** [Mindwerks/worldengine#180]
To do so we should
* [ ] find out which format to support
* [ ] implement an operation to export generated worlds to that format
* [ ] describe in the manual a scenario usage

**Comment by ftomassetti** [comment]
The format which is best to support is TMX and we can use a python library for that: http://pytmxlib.readthedocs.org/en/latest/

**Comment by ftomassetti** [comment]
First experiments

![screenshot from 2015-11-03 23 32 10](https://cloud.githubusercontent.com/assets/439078/10923820/593323e0-8283-11e5-94dd-f1a81579742d.png)

**Comment by psi29a** [comment]
Nice!

I can see this as being useful for game development. :)

**Comment by ftomassetti** [comment]
Indeed: the format is used by so many libraries for all sort of languages so many games would be able to use our maps out of the box. In addition to that users get an editor to customize the map we generate. For example they could add cities and change the shape of a forest or add their own touch to the generated world.

I really think we should push on interoperability and finding tools and scenarios which can leverage WorldEngine.

**Comment by Asday** [comment]
Is this issue dead?  I would love to see it completed.

**Comment by psi29a** [comment]
Me too! :)

**Comment by ftomassetti** [comment]
This issue was implemented, but it was not published, I should dig that old code... it would help if someone volunteer to help me polishing :)

**Comment by Asday** [comment]
If you have time to mentor me over the weekend, I can.  I'm a lowly web developer by trade, though, so I may be a little slow to start.

**Comment by ftomassetti** [comment]
I would be happy to, you can reach me at any time at federico _at_ tomassetti.me or in this issue

**Comment by ftomassetti** [comment]
Happy you find it useful!

**Comment by DblK** [comment]
Regarding the exporter, I do have question about the generated world.
`Ocean` have boolean, but `plates`, `water` have numbers from 0 to 1 and `elevation` have integer from 0 to 12 (??).
I can not find in the doc any explanations. Can you point me somewhere so I can understand which array I should take into account to generate the map.
Any clues will be appreciated ^^

**Comment by ftomassetti** [comment]
I am sorry but I have not touched the project in years, so I am afraid I cannot help. The project is unmaintained
